CHANGES.md

```diff
@@ -1,9 +1,25 @@
 # Changes
 
 ### v0.4.6
-* Refactoring of metrics:
+* Refactor metrics:
   * Remove `GroupMetricResult` type in favour of a `Bunch`.
```

Comment:
nit: if you make other changes as well can we make this favor instead of favour?

Reply:
otherwise ignore

Reply:
done